src/pudl/transform/nrelatb.py

```diff
@@ -121,12 +121,13 @@ def transform_unstack(
 
     """
     nrelatb_unstacked = (
-        nrelatb[
-            nrelatb.core_metric_parameter.isin(table_unstacker.core_metric_parameters)
+        nrelatb.loc[
+            nrelatb.core_metric_parameter.isin(table_unstacker.core_metric_parameters),
+            table_unstacker.idx + ["value"],
         ]
-        .set_index(table_unstacker.idx)
-        .sort_index()[["value"]]
         .drop_duplicates()
+        .set_index(table_unstacker.idx)
+        .sort_index()
```

Comment:
i was previously applying the `drop_duplicates` after the setting of the index so it was dropping any duplicate `value`. so now i'm selecting only the idx + value columns first, then drop_dup-ing, then index-ing.

Reply:
Ahh makes sense. Should we have some sort of "there's X% non-null values" check for this table?